OpenRA.Mods.Common/Traits/ProductionParadrop.cs

```diff
@@ -76,7 +75,7 @@ public override bool Produce(Actor self, ActorInfo producee, string factionVaria
 					new FacingInit(64)
 				});
 
-				actor.QueueActivity(new Fly(actor, Target.FromCell(w, dropPos)));
+				actor.QueueActivity(new Fly(actor, Target.FromActor(self)));
```

Comment:
I'm not quite sure if there is all that much difference between FromCell and FromPos. The problem is that by doing anything based on the actor's location, it's going to be outdated almost instantly if the actor is moving. Really what needs to be done is pass the plane somehow to the doProduction method.